.circleci/config.yml

```diff
@@ -61,6 +61,10 @@ jobs:
       - attach_workspace:
           at: *WORKSPACE
       - checkout
+      - run:
+        name: "Setup GraphQL endpoint"
+        command: |
+          echo 'export VUE_APP_GRAPHQL_URL="https://production-api.lunie.io/"' >> $BASH_ENV
```

Comment:
this should come from an environment variable set in CircleCI not in plain text.

Reply:
the build is actually not triggered here. it is in Netlify. There this variable is already set. Why did you want to set it for testing?

Reply:
Yes, I already realized these commits have been of little use :woman_shrugging: